master/getting-started/kubernetes/index.md

```diff
@@ -1,41 +1,195 @@
+   ```
+   
+1. Execute the following commands to configure kubectl (also returned by
+   `kubeadm init`).
+
+   ```
+   mkdir -p $HOME/.kube
+   sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+   sudo chown $(id -u):$(id -g) $HOME/.kube/config
+   ```
+   
+1. Install Calico, a pod network, and a single node etcd with the 
+   following command.
+
+   ```
+   kubectl apply -f \
+   https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml
+   ```
+   
+   > **Note**: You can also 
+   > [view the YAML in your browser](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml){:target="_blank"}.
+   {: .alert .alert-info}
+
+   You should see the following output.
+
+   ```
+   configmap "calico-config" created
+   daemonset "calico-etcd" created
+   service "calico-etcd" created
+   daemonset "calico-node" created
+   deployment "calico-kube-controllers" created
+   deployment "calico-policy-controller" created
+   clusterrolebinding "calico-cni-plugin" created
+   clusterrole "calico-cni-plugin" created
+   serviceaccount "calico-cni-plugin" created
+   clusterrolebinding "calico-kube-controllers" created
+   clusterrole "calico-kube-controllers" created
+   serviceaccount "calico-kube-controllers" created
+   ```
+   
+1. Confirm that all of the pods are running with the following command.
+
+   ```
+   kubectl get pods --all-namespaces
+   ```
+   
+   Review the output to ensure that each pod has the `STATUS` of `Running`.
+
+   ```
+   NAMESPACE    NAME                                       READY  STATUS   RESTARTS  AGE
+   kube-system  calico-etcd-x2482                          1/1    Running  0         2m
+   kube-system  calico-kube-controllers-6ff88bf6d4-tgtzb   1/1    Running  0         2m
+   kube-system  calico-node-24h85                          2/2    Running  0         2m
+   kube-system  etcd-jbaker-virtualbox                     1/1    Running  0         6m
+   kube-system  kube-apiserver-jbaker-virtualbox           1/1    Running  0         6m
+   kube-system  kube-controller-manager-jbaker-virtualbox  1/1    Running  0         6m
+   kube-system  kube-dns-545bc4bfd4-67qqp                  3/3    Running  0         5m
+   kube-system  kube-proxy-8fzp2                           1/1    Running  0         5m
+   kube-system  kube-scheduler-jbaker-virtualbox           1/1    Running  0         5m
+   ```
+   
+   > **Note**: You may need to give the services a little time to come
+   up. Continue issuing the `kubectl get pods --all-namespaces` command
+   until you see each one achieve a `STATUS` of `Running`. 
+   {: .alert .alert-info}
+
+1. Remove the taints on the master so that you can schedule pods
+   on it.
+   
+   ```
+   kubectl taint nodes --all node-role.kubernetes.io/master-
+   ```
+
+   It should return the following.
+
+   ```
+   node "<your-hostname>" untainted
+   ```
+   
+1. Switch to a root shell.
```

Comment:
Why do we do this rather than just run the following commands as `sudo`?

Reply:
It is a great question and I can't understand it either, but that's how they had it on the k8s doc site. It is definitely awkward to have to  do this.

Reply:
I have put it on my list of follow-up items.